packages/ui-kit/src/index.css

```diff
@@ -0,0 +1,9 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;600&display=swap");
```

Comment:
Worth considering in the future: self-hosting the fonts
https://insights.project-a.com/google-fonts-breaches-gdpr-1ae823dba41e

Reply:
This `@import` is only used in Storybook Preview, it's not added to build :)